### DISCUSSION TITLE
[Community post] Fuse problems? ERHQ

### DISCUSSION THREAD

**Discussion by pitylee** [raomin/ESPAltherma#233] [Q&A]
Hello community,

Did you guys have problem with fuses on outdoor units? It keeps blowing the F1A ceramic 6.3x32mm fuses, and showing U4 error, and have no idea why it does that.

**Comment by raomin** [comment]
I'll convert this one to a discussion as it's not really an ESPAltherma issue.

**Comment by Tardymo** [comment]
Is that a 3-phase outdoor unit you are talking about?

**Comment by pitylee** [comment]
yes, it has a bunch of fuses that are not even on the technical drawings. after all, it got shorted, and the PCB needed some transistors replaced.
